msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AbstractClientApplicationBase.java

```diff
@@ -673,7 +677,7 @@ public T azureRegion(String val) {
         yet still want MSAL to accept any authority that you will provide,
         you can use a ``False`` to unconditionally disable Instance Discovery. */
         public T instanceDiscovery(boolean val) {
```

Comment:
Should not be part of the Managed Identity apis... Not sure if it is or not.

Reply:
It is not. This is part of Abstract class. Will be taken care of in refactoring